google/resource_storage_bucket_iampolicy.go

```diff
@@ -0,0 +1,122 @@
+package google
```

Comment:
This is definitely a nitpick, but I'd love to see a `_` between `iam` and `policy` in the filename here.